compiler/cli.py

```diff
@@ -23,6 +23,17 @@
 from subpar.compiler import python_archive
 
 
+def bool_from_string(raw_value):
+    """Parse a boolean command line argument value"""
+    if raw_value == 'True':
+        return True
+    elif raw_value == 'False':
+        return False
+    else:
+        raise argparse.ArgumentTypeError(
```

Comment:
I'm guessing this will never get an actual `bool`?

Reply:
Yes, the behavior here is a little surprising.  The value always starts as a string, and if you do `bool("any string")` you always get `True` as the result.  In particular, `bool("False")` returns `True`.  There's no convenient builtin to parse a bool from a string, otherwise I would use it here.